.github/conversational/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+* **Please check if the PR fulfills these requirements**
+- [ ] The commit message follows our guidelines
+- [ ] Unit Tests for the changes that have been added (for bug fixes / features)
+- [ ] Docs have been added / updated (for bug fixes / features)
+
+
+* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
+
+
+
+* **What is the current behavior?** (You can also link to an open issue here)
+
+
+
+* **What is the new behavior (if this is a feature change)?**
+
+
+
+* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
```

Comment:
Not sure if this line should be included. @pythonindia/pycon-2018-team Any comment on this line ?

Reply:
We can remove it. I don't know how to proceed with this message.